### PR TITLE
add filler for codeCopyOffset, change location of the test to stMemoryTest

### PR DESCRIPTION
**Pull request by jwasinger**
Corrects an error with this PR: https://github.com/ethereum/tests/pull/382

### CODE REVIEW

File: GeneralStateTests/stMemoryTest/codeCopyOffset.json [hunk 1]
```diff
@@ -45,7 +89,7 @@
             },
             "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" : {
                 "balance" : "0x0de0b6b3a7640000",
-                "code" : "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff600052601061ffff61010037600051600055",
+                "code" : "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff600052601061ffff600039600051600055",
```
Comment (pirapira) [inline GeneralStateTests/stMemoryTest/codeCopyOffset.json:92]: I feel like a detective here.

Is this change intended?  Even if unintended, it's fine if it's harmless.

File: GeneralStateTests/stMemoryTest/codeCopyOffset.json [hunk 2]
```diff
@@ -2,10 +2,10 @@
     "codeCopyOffset" : {
         "_info" : {
             "comment" : "",
-            "filledwith" : "cpp-1.3.0+commit.dda985df.Linux.g++",
+            "filledwith" : "cpp-1.3.0+commit.97bd859a.Linux.g++",
             "lllcversion" : "Version: 0.4.14+commit.c2215d46.Linux.g++",
-            "source" : "src/GeneralStateTestsFiller/stCodeCopyTest/codeCopyOffsetFiller.json",
```
Comment (winsvega) [inline GeneralStateTests/stMemoryTest/codeCopyOffset.json:7]: you have to remove filler and filled test from that folder as well.
try running 
`./testeth -t GeneralStateTests`  
'./testeth -t BCGeneralStateTests'
and see that there are no errors

File: GeneralStateTests/stMemoryTest/codeCopyOffset.json (hunk 1 above)
Comment (jwasinger) [reply GeneralStateTests/stMemoryTest/codeCopyOffset.json:92]: No.  it was unintended.  But you're right, it shouldn't matter for this test case (as long as source offset > source size, and the amount of data being written > 0)
